internal/bgp/frr/config.go

```diff
@@ -36,7 +37,16 @@ router bgp {{.MyASN}}
 {{- end }}
 {{range .Neighbors }}
   neighbor {{.Addr}} remote-as {{.ASN}}
+  {{ if .Port }}
```

Comment:
```suggestion
  {{ if .Port -}}
  neighbor {{.Addr}} port {{.Port}}
  {{- end}}
  neighbor {{.Addr}} timers {{.KeepaliveTime}} {{.HoldTime}}
  {{- if .Password }}
  neighbor {{.Addr}} password {{.Password}}
  {{end -}}
  {{if .SrcAddr -}}
  neighbor {{.Addr}} update-source {{.SrcAddr}}
  {{- end}}
```

I'd change the template to fix whitespaces when the value is not there